src/plugins/explore/public/components/chart/timechart_header/timechart_header.test.tsx

```diff
@@ -216,7 +216,10 @@ describe('timechart header', function () {
 
     it('should render BreakdownFieldSelector when shouldShowBreakdownSelector returns true and services is provided', () => {
       mockShouldShowBreakdownSelector.mockReturnValue(true);
-      const mockServices = { data: {} };
+      const mockServices = {
+        data: {},
+        uiSettings: { get: jest.fn().mockReturnValue(true) },
```

Comment:
nit: it can be nice to mock the specific FF you want, just to make things easier for the next person. even though the behavior is the same, it self-documents to let people know which specific FF values are needed for the tests to pass.

Reply:
I'll change this to mock the feature flag